pom.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>strike-off-objections-api</artifactId>
+    <version>unversioned</version>
+    <packaging>jar</packaging>
+
+    <name>strike-off-objections-api</name>
+    <description>Create a new strike-off objection request</description>
+
+    <parent>
+        <groupId>uk.gov.companieshouse</groupId>
+        <artifactId>companies-house-spring-boot-parent</artifactId>
+        <version>1.3.0</version>
```

Comment:
check this is the latest parent and latest ch dependencies. i know for PTF we weren't on the latest structured logging version

Reply:
Yes, this is the latest parent but some of the other dependencies were older. I've updated them and app still builds/starts up/works.

Have noticed that the logging output now looks a bit strange (just more verbose). Might be possible to tweak this, so will investigate but plan to change (if possible) in a separate PR.